torch/_inductor/utils.py

```diff
@@ -217,29 +229,6 @@ def print_performance(fn, args=(), times=10, repeat=10, baseline=1.0):
 immutable_list.__hash__ = lambda self: hash(tuple(self))
 
 
-def freeze_inputs(f):
-    """
-    Useful for wrapping lists in tuples for caching purposes
-    """
-
-    def freeze_value(x):
-        if isinstance(x, (immutable_dict, immutable_list)):
-            return x
-        if isinstance(x, list):
-            return immutable_list(x)
-        if isinstance(x, dict):
-            return immutable_dict(x)
-        return x
-
-    @functools.wraps(f)
-    def wrapped(*args):
-        args = [freeze_value(x) for x in args]
-        return f(*args)
-
-    wrapped.cache_info = f.cache_info
```

Comment:
This line was causing a type error as mypy thinks `f` is a simple Callable. Since `freeze_inputs` doesn't seem to be used anywhere in the codebase, I opted to remove it instead of fixing this issue.